pkg/settings/settings.go

```diff
@@ -18,6 +20,8 @@ var (
 	APIUIVersion           = NewSetting("api-ui-version", "1.1.9") // Please update the HARVESTER_API_UI_VERSION in package/Dockerfile when updating the version here.
 	AuthTokenMaxTTLMinutes = NewSetting("auth-token-max-ttl-minutes", "720")
 	AuthSecretName         = NewSetting("auth-secret-name", "harvester-key-holder")
+	NoDefaultAdmin         = NewSetting("no-default-admin", "")
```

Comment:
why do we need this setting?

Reply:
this is for when user don't want to create default admin, inspired by rancher, for example, when user just enable k8s mode, no need default admin.